airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/ctid/CtidUtilsTest.java

```diff
@@ -31,7 +32,7 @@
 import java.util.List;
 import org.junit.jupiter.api.Test;
 
-public class XminCtidUtilsTest {
+public class CtidUtilsTest {
```

Comment:
can we add a test that exercises the stream categorization for standard (cursor-based) syncs?